### PR TITLE
update doc

### PR DESCRIPTION
**Pull request by KirillZubov**

### CODE REVIEW

File: docs/src/solvers/pinn.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ discretization = PhysicsInformedNN(chain,
                                    nothing; #init_params
                                    phi = nothing,
                                    derivative = nothing,
-                                   strategy = GridTraining())
+                                   strategy = nothing)
```
Comment (ChrisRackauckas) [inline docs/src/solvers/pinn.md:29]: hmm, if strategy is required, shouldn't it be a required argument?

Comment (KirillZubov) [reply docs/src/solvers/pinn.md]: yeap, you right

File: src/pinns_pde_solve.jl [hunk 2]
```diff
@@ -20,20 +21,20 @@ struct PhysicsInformedNN{C,P,PH,DER,T,K}
 end
 
 function PhysicsInformedNN(chain,
-                           init_params = nothing;
+                           strategy;
+                           _initθ = nothing,
```
Comment (ChrisRackauckas) [inline src/pinns_pde_solve.jl:25]: Keep the old name. We shouldn't require Unicode in the API otherwise some terminals won't be able to use the argument.

Comment (KirillZubov) [reply src/pinns_pde_solve.jl:25]: Actually, it already was `initθ` in arguments of PhysicsInformedNN. Ok, I will rename all the `initθ` to `init_params`.

Comment (ChrisRackauckas) [reply src/pinns_pde_solve.jl:25]: Yeah internally is fine, but we have a policy to not expose Unicode externally.

Comment (KirillZubov) [reply src/pinns_pde_solve.jl:25]: ok

**Comment by ChrisRackauckas** [comment]
Hmm tests are failing?

**Pull request merged**